Manifests/ManagedPreferencesApplications/com.github.macadmins.Nudge.plist

```diff
@@ -161,60 +188,271 @@
+					<string>attemptToFetchMajorUpgrade</string>
+					<key>pfm_title</key>
+					<string>Attempt to Fetch Major Upgrade</string>
+					<key>pfm_type</key>
+					<string>boolean</string>
+				</dict>
+				<dict>
+					<key>pfm_app_min</key>
+					<string>1.1.7</string>
+					<key>pfm_description_reference</key>
+					<string>The application Bundle ID which Nudge allows without taking focus. (You can specify one or more Bundle ID.)
+
+Disables Nudge's re-activation logic for the listed applications. If one of the specified applications is foremost, Nudge will not reactivate itself. (This feature does not work for Nudge's initial activation.).</string>
+					<key>pfm_documentation_url</key>
+					<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#acceptableapplicationbundleids---type-array-default-value--required-no</string>
+					<key>pfm_name</key>
+					<string>acceptableApplicationBundleIDs</string>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_name</key>
+							<string>acceptableApplicationBundleID</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+					</array>
+					<key>pfm_title</key>
+					<string>Acceptable Application Bundle IDs</string>
+					<key>pfm_type</key>
+					<string>array</string>
+				</dict>
+				<dict>
+					<key>pfm_app_min</key>
+					<string>1.1.7</string>
+					<key>pfm_default</key>
+					<string>false</string>
+					<key>pfm_description_reference</key>
+					<string>When enabled, Nudge will not activate or re-activate when assertions are currently set.</string>
+					<key>pfm_documentation_url</key>
+					<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#acceptableassertionusage---type-boolean-default-value-false-required-no</string>
+					<key>pfm_name</key>
+					<string>acceptableAssertionUsage</string>
+					<key>pfm_title</key>
+					<string>Acceptable Assertion Usage</string>
+					<key>pfm_type</key>
+					<string>boolean</string>
+				</dict>
+				<dict>
+					<key>pfm_app_min</key>
+					<string>1.1.7</string>
+					<key>pfm_description_reference</key>
+					<string>The application names using assertions which Nudge allows without taking focus. You can specify one or more applications. To find the names please run /usr/bin/pmset -g assertions in Terminal while the application is open and running.</string>
+					<key>pfm_documentation_url</key>
+					<string>https://github.com/macadmins/nudge/wiki/optionalFeatures#acceptableassertionapplicationnames---type-array-default-value--required-no</string>
+					<key>pfm_name</key>
+					<string>acceptableAssertionApplicationNames</string>
+					<key>pfm_title</key>
+					<string>Acceptable Assertion Application Names</string>
+					<key>pfm_subkeys</key>
+  					<array>
+    					<dict>
+							<key>pfm_name</key>
+							<string>acceptableAssertionApplicationName</string>
+							<key>pfm_title</key>
+							<string>Acceptable Assertion Application Name</string>
```

Comment:
Same as previous comment, no `pfm_name` should be here when the main preference is an array.  `pfm_title` is all that's needed besides `pfm_type`